app/utils/file_utils.py

```
@@ -0,0 +1,35 @@
+def download_pdf_from_s3(url: str, timeout: int = 30) -> str:
+    """
+    안전하게 PDF를 다운로드하여 임시 파일 경로 반환.
+    Caller is responsible for deleting the file when done.
+    """
+    if not url.lower().startswith(("http://", "https://")):
+        raise HTTPException(status_code=400, detail="Invalid URL")
+
+    try:
+        with requests.get(url, stream=True, timeout=timeout) as r:
+            r.raise_for_status()
+            content_type = r.headers.get("content-type", "")
```

Ruff (0.13.1)

Local variable `content_type` is assigned to but never used

Remove assignment to unused variable `content_type`

(F841)